compliance_checker/cf/util.py

```diff
@@ -327,15 +330,25 @@ def download_cf_standard_name_table(version, location=None):
     if location is None:  # This case occurs when updating the packaged version from command line
         location = resource_filename('compliance_checker', 'data/cf-standard-name-table.xml')
 
-    url = "http://cfconventions.org/Data/cf-standard-names/{0}/src/cf-standard-name-table.xml".format(version)
-    r = requests.get(url, allow_redirects=True)
-    if r.status_code == 200:
-        print("Downloading cf-standard-names table version {0} from: {1}".format(version, url), file=sys.stderr)
-        with open(location, 'wb') as f:
-            f.write(r.content)
+    if version == 'latest':
+        tables_tree = lxml.html.parse("http://cfconventions.org/documents.html")
+        end_str = "cf-standard-name-table.xml"
+        xpath_expr = ("//a[substring(@href, string-length(@href) - "
```

Comment:
NB: However verbose this is, this really is how you do something equivalent to something like Python's `str.endswith` when using XPath 1.0.  XPath 2.0 has an `ends-with` function, which is considerably more concise, but LXML only uses XPath 1.0.